lighthouse-core/lib/i18n/i18n.js

```diff
@@ -130,10 +130,61 @@ function lookupLocale(locale) {
  * @param {Record<string, string | number>} [values]
  */
 function _preprocessMessageValues(icuMessage, values = {}) {
-  const clonedValues = JSON.parse(JSON.stringify(values));
   const parsed = MessageParser.parse(icuMessage);
+
+  const elements = _collectAllCustomElementsFromICU(parsed.elements);
+
+  return _processParsedElements(Array.from(elements.values()), JSON.parse(JSON.stringify(values)));
```

Comment:
sorry if my comment was misleading, I actually like the `clonedValues` as an explicit variable as on current master (the name helps explain what it's up to with the JSON.parse(stringify)), just not as a `let` and redefined as in the previous commit